### PR TITLE
Added refocus function

### PR DESCRIPTION
**Pull request by DonovonJenson**

### CODE REVIEW

**Comment by DonovonJenson** [comment]
Tracks the longest focus interval in a session and all refocus timestamps

File: src/app.jsx [hunk 1]
```diff
@@ -45,6 +47,10 @@ class App extends React.Component {
       this.setState({countdown: this.state.sessionLength});
     }
 
+    if (!this.state.focusTimestamp) {
```
Comment (aploftus) [inline src/app.jsx]: at start of function, reset the focusTimestamp to empty array, reset the refocusCount to 0

File: src/app.jsx [hunk 2]
```diff
@@ -62,7 +68,13 @@ class App extends React.Component {
   }
 
   refocus() {
+    //Need to refactor focus count to just print length of focus
```
Comment (aploftus) [inline src/app.jsx:71]: Great, added to backlog

**Review by aploftus — COMMENTED** [review]
Please fix startCountdown method

**Pull request merged**